### PR TITLE
Fix account info

### PR DESCRIPTION
**Pull request by iamajvillalobos**
## Description
This is a fix for the account info not working as expected. It now returns a `Stellar::AccountInfo` object.

## Issue
[account_info fetch does not work](https://github.com/stellar/ruby-stellar-sdk/issues/14)

### CODE REVIEW

File: lib/stellar/account_info.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+require 'toml-rb'
+require 'uri'
+
+module Stellar
+  class AccountInfo
```
Comment (nullstyle) [inline lib/stellar/account_info.rb]: I don't think the introduction of this class is super useful at this point.  Keeping this up to date will be a pain and it doesn't give much benefit.

File: lib/stellar/client.rb [hunk 2]
```diff
@@ -46,11 +46,11 @@ def friendbot(account)
       raise NotImplementedError
     end
 
-    # Contract Stellar::Account => Stellar::AccountInfo
-    Contract Stellar::Account => Any
+    Contract Stellar::Account => Stellar::AccountInfo
     def account_info(account)
       account_id  = account.address
-      @horizon.account(account_id:account_id)
+      resource = @horizon.account(account_id:account_id)._get
```
Comment (nullstyle) [inline lib/stellar/client.rb]: Instead of introducing `AccountInfo`, how about we just return the result of `#_get` and call it a day?

**Review by nullstyle — CHANGES_REQUESTED** [review]
Hi @iamarmanjon,

Thanks for the contributions.  Have you signed our contributor agreement?  https://docs.google.com/forms/d/1g7EF6PERciwn7zfmfke5Sir2n10yddGGSXyZsq98tVY/viewform?usp=send_form

Thanks!

File: lib/stellar/client.rb (hunk 2 above)
Comment (iamajvillalobos) [reply lib/stellar/client.rb]: I was gonna do that first, but I saw a commented out `Contract Stellar::Account => Stellar::AccountInfo` :), but I could get remove that.

Comment (nullstyle) [reply lib/stellar/client.rb]: :)  Yeah, I think I was getting ahead of myself with the comment line.  Please remove the type, and we'll add it back when we specifically need it. 

Thanks!

**Comment by iamajvillalobos** [comment]
Hi @nullstyle, I made the changes you requested :)

**Review by nullstyle — APPROVED** [review]

**Comment by nullstyle** [comment]
Beautiful!  thanks very much @iamarmanjon

**Pull request merged**